### DISCUSSION TITLE
Practica HTML duda sobre parte del Pastelero

### DISCUSSION THREAD

**Discussion by RaulGuereca** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#247] [Q&A]
ALguien me podria decir a nivel general que es lo que hay que hacer en la parte referente al pastelero del ejercicio de HTML. no entiendo lo que hay que hacer en esa parte? agradezco su ayuda

**Comment by JuanSalvador73** [comment]
1.- Hacer un logo para tu pastelería - muchos han usado imágenes editadas de internet.

2.- Hacer un archivo html donde hables de tu pastelería, slogan. Básicamente es SOLO el html, solo documento simple como se vió en uno de los live.

3.- En ese HTML vas a incluir datos que le pedirás al cliente como va a ser su pedido, por medio de un formulario. Pedirá sabor, adornos, etc. También los datos del cliente como nombre, teléfono, y los horarios de la pastelería.

4.- Hacer un html para el pastelero, donde pondrás los datos que escogió para hacer el pastel  que le pidió el cliente, usa una tabla , como sabor, adornos, etc. Así como quién lo pidió, y a donde se entrega si es el caso.

BONUS:

Si tienes lo anterior, también puedes publicarlo - aquí en Git por ejemplo - y poner el link. También puedes agregarle CSS para que se vea mas estético.

Estoy prácticando en un sitio dummy aparte de Gatos. Lo que llevo hasta ahora es un HTML simple - como lo que hay que entregar: 

![image](https://user-images.githubusercontent.com/99113565/156849818-015507e4-697f-415b-aaca-f30cdfc9591a.png)

**Comment by doguedogue** [comment]
Es como una parte administrativa, donde él pastelero (administrador) puede revisar:
- La cantidad de sabores que le quedan para hacer los pasteles.
- La cantidad de adornos que le quedan para hacer los pasteles.
1.           Este requiere como un inventario y llevar un control de sus ingredientes faltantes.
2.           Todo es simulado ya que para llevar un control deberías tener una base de datos o algo parecido.

- Tabla donde aparezcan los pedidos que se han realizado con los datos del formulario.
1.           Algo así como la parte del abogabot que los abogados iban a ver quién pagó el servicio de la demanda.

**Reply by GloriaCoralCerecedo** [reply]
En esos casos y en el de la tabla, se pueden meter información manual como ejemplo, me imagino.

**Reply by BattiestWinter** [reply]
> En esos casos y en el de la tabla, se pueden meter información manual como ejemplo, me imagino.

Si, bueno yo metí registros de ejemplos jaja, porque lo común sería obtener esa información de la base de datos
